modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -400,7 +400,8 @@ public List<ServiceRegistration<?>> deploy(
 				if (Objects.equals(
 						objectRelationship.getType(),
 						ObjectRelationshipConstants.TYPE_MANY_TO_MANY) &&
-					!objectRelationship.isReverse()) {
+					(objectRelationship.getObjectDefinitionId1() ==
+						objectDefinition.getObjectDefinitionId())) {
```

Comment:
@jkappler 

`objectRelationship.getObjectDefinitionId1() == objectDefinition.getObjectDefinitionId()`

it will be always true as [this method](https://github.com/liferay-objects/liferay-portal/pull/1797/commits/ae61e44e43d73096436a6eb6abadc21f1cb27c0e#diff-ee49a8b10160e96b173fd6482ebe52fdf31c2bbc9e4babd70de4e4e29ad70bc3R395-R396) fin by objectDefinitionId1 and it receives objectDefinition.getObjectDefinitionId() as parameter.

Reply:
we can just remove it and keep:

`Objects.equals(objectRelationship.getType(), ObjectRelationshipConstants.TYPE_MANY_TO_MANY)`